http-client/src/main/java/io/airlift/http/client/jetty/DefaulHttpClientLogger.java

```diff
@@ -49,10 +49,15 @@
         TimeBasedRollingPolicy<HttpRequestEvent> rollingPolicy = new TimeBasedRollingPolicy<>();
 
         rollingPolicy.setContext(context);
-        rollingPolicy.setFileNamePattern(filename + "-%d{yyyy-MM-dd}.%i.log.gz");
         rollingPolicy.setMaxHistory(maxHistory);
         rollingPolicy.setTimeBasedFileNamingAndTriggeringPolicy(triggeringPolicy);
         rollingPolicy.setParent(fileAppender);
+        if (compressionEnabled) {
```

Comment:
Should we avoid repeating the pattern and make it clear this is appending an extension?
```java
rollingPolicy.setFileNamePattern(filename + "-%d{yyyy-MM-dd}.%i.log");
if (compressionEnabled) {
    rollingPolicy.setFileNamePattern(rollingPolicy.getFileNamePattern() + ".gz");
}

Reply:
I am fine either way, updating.